src/plugins/discover/public/application/components/default_discover_table/_doc_table.scss

```diff
@@ -69,6 +69,23 @@ doc-table {
   }
 }
 
+.osdDiscoverTable {
+  @include ouiCodeFont;
+
+  & > tbody > tr:not(.osdDiscoverInViewRow):not(.osdDiscoverExpandedRow) {
+    visibility: hidden;
+  }
```

Comment:
Do we still need this?

Reply:
Yep

Reply:
hmm, I dont see any other mention of the `osdDiscoverInViewRow` selector. Where is it added?

Reply:
OH! you don't need it! pff.

Reply:
dropped